app/views/components/_navigation-sidebar.html.erb

```diff
@@ -0,0 +1,48 @@
+<%
+  related_items ||= []
+  topics ||= []
+  publishers ||= []
+  collections ||= []
+  policies ||= []
+  related_content = [
+    {"related_items" => related_items},
+    {"topics" => topics},
+    {"publishers" => publishers},
+    {"collections" => collections},
+    {"policies" => policies}
+  ]
+%>
+<% if related_content.map(&:values).flatten.any? %>
+  <aside class="app-c-navigation-sidebar" role="complementary">
+    <% related_content.each do |section| %>
+      <% section.select do |section_title, links| %>
+
```

Comment:
Lines of whitespace look a little random.

Reply:
Hmm, I put these in to break up the code into the 2 separate `if` statements, as otherwise I found it a bit difficult to read/follow the logic. Happy to remove if you think it's fine without the spacing though

Reply:
I think it's ok without. If you do leave it in then I'd make it more symmetrical so you have an opening and a closing space around the thing you're making more readable.